.dbeaver/data-sources.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Was this committed intentionally?

Reply:
Removed. Was missing in `.gitignore`. Disposable local database, so no harm done.